lib/helpers.js

```diff
@@ -0,0 +1,246 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const base58 = require('bs58');
+const bedrock = require('bedrock');
+const brZCapStorage = require('bedrock-zcap-storage');
+const {config, util: {BedrockError}} = bedrock;
+const {promisify} = require('util');
+const crypto = require('crypto');
+const getRandomBytes = promisify(crypto.randomBytes);
+const jsigs = require('jsonld-signatures');
+const {SECURITY_CONTEXT_V2_URL} = jsigs;
+const {Ed25519Signature2018} = jsigs.suites;
+const storage = require('./storage');
+const {verifyCapabilityInvocation} = require('http-signature-zcap-verify');
+
+const routes = {
+  edvs: '/edvs',
+};
+
+const {baseUri} = config.server;
+const baseStorageUrl = `${baseUri}${routes.edvs}/`;
+
+// TODO: some of the following code is a target for reusability in other
+// modules and should be factored out for reuse
+
+// Note: for dereferencing `did:key` URLs
+exports.DOCUMENT_LOADER = async url => {
+  // TODO: move to did-key lib
+  if(url.startsWith('did:key:')) {
```

Comment:
This doesn't look like it has the changes on master, might need a rebase or an update since things were moved to a new file.

Reply:
Fixed.